src/core/layertree/qgslayertreemodellegendnode.h

```diff
@@ -162,6 +162,11 @@ class CORE_EXPORT QgsSymbolV2LegendNode : public QgsLayerTreeModelLegendNode
 
     virtual void invalidateMapBasedData() override;
 
+    void setIconSize( const QSize& sz ) { mIconSize = sz; }
+    QSize iconSize() const { return mIconSize; }
+
+    void setCrop( bool isCropRequired ) { mCrop = isCropRequired; }
```

Comment:
I'd suggest adding a getter for mCrop too. There's nothing more frustrating then a missing getter when your project depends on it ;)